arm/Microsoft.ManagedServices/registrationDefinitions/deploy.bicep

```diff
@@ -43,6 +43,11 @@ module registrationAssignment_rg '.bicep/nested_registrationAssignment.bicep' =
   }
 }
 
+@description('The name of the registration definition')
 output registrationDefinitionName string = registrationDefinition.name
-output registrationDefinitionId string = registrationDefinition.id
+
+@description('The resource ID of the registration definition')
+output registrationDefinitionResourceId string = registrationDefinition.id
+
+@description('The registration assignment resource ID')
```

Comment:
Is this output needed and should we add the resource group as per all other modules?

Reply:
Added the subscriptionName. ResourceGroup does not work in this case as the scope is subscription.

Regarding the other output I'd suggest to tackle this in another PR